tests/failpoints/cases/test_region_cache_engine.rs

```diff
@@ -799,3 +799,47 @@ fn test_delete_range() {
     delete_range(false);
     delete_range(true);
 }
+
+#[test]
+fn test_evict_on_flashback() {
+    let mut cluster = new_server_cluster_with_hybrid_engine_with_no_region_cache(0, 1);
+    cluster.cfg.raft_store.apply_batch_system.pool_size = 1;
+    cluster.run();
+
+    let table = ProductTable::new();
+
+    must_copr_load_data(&mut cluster, &table, 1);
+    must_copr_load_data(&mut cluster, &table, 2);
+    must_copr_load_data(&mut cluster, &table, 3);
+
+    {
+        let region_cache_engine = cluster.sim.rl().get_region_cache_engine(1);
+        // Load the whole range as if it is not splitted. Loading process should handle
+        // it correctly.
```

Comment:
Wrong comments? This case is not about split.

Reply:
Yes, I will remove it.